streaming/index.js

```diff
@@ -29,25 +29,9 @@ log.level = process.env.LOG_LEVEL || 'verbose';
  */
 const redisUrlToClient = async (defaultConfig, redisUrl) => {
```

Comment:
we could also change this method to just be `redisClient = async (environment) => {` and do `const { redisParams, redisUrl, redisPrefix } = redisConfigFromEnv(environment);` inside this function.

Then the two lines at ~170/190, become:

```diff
-  const redisSubscribeClient = await redisUrlToClient(redisParams, redisUrl);
-  const redisClient = await redisUrlToClient(redisParams, redisUrl);
+  const redisSubscribeClient = await redisClient(process.env);
+  const redisClient = await redisClient(process.env);
```